### PR TITLE
Fix bugs

### PR DESCRIPTION
**Pull request by AnthonyAmanse**
Part A reference to Istio release root directory

### CODE REVIEW

**Comment by Tomcli** [comment]
Make sure you change the Travis link on your README

File: microservices/ratings/ratings.js [hunk 1]
```diff
@@ -94,11 +94,11 @@ dispatcher.onGet("/ratings", function(req, res) {
         ratingsResponse = {"Reviewer1": first_rating, "Reviewer2": second_rating, "Reviewer3": third_rating, "Reviewer4": fourth_rating, "Reviewer5": fifth_rating};
         console.log(ratingsResponse);
         json = JSON.stringify(ratingsResponse)
+        res.writeHead(200, {"Content-type": "application/json"})
+        res.end(json)
```
Comment (animeshsingh) [inline microservices/ratings/ratings.js]: What bug is this targeting?

File: microservices/ratings/ratings.js [hunk 2]
```diff
@@ -94,11 +94,11 @@ dispatcher.onGet("/ratings", function(req, res) {
...
 
     });
     connection.end();
-    res.writeHead(200, {"Content-type": "application/json"})
-    res.end(json)
 })
```
Comment (animeshsingh) [inline microservices/ratings/ratings.js]: Is it targeting any particular open issues? i believe we are closing the connection now after returning the response

Comment (AnthonyAmanse) [inline microservices/ratings/ratings.js]: No open issues for this one but based on my experience, this was more stable.

**Comment by Tomcli** [comment]
LGTM

**Pull request merged**